CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# v1.1.0 2018-05-23
```

Comment:
5/23? Assuming this is otherwise ready to go, shouldn't the date be today?

Reply:
Yup! Fixed.